CONTRIBUTING.rst

```diff
@@ -86,93 +83,108 @@ An Example
+branch, there exists your fork's copy of the ``main`` and ``work`` branches,
+*AND* there exists your *local* copy of the ``main`` and ``work`` branches. 
 
-Workflow: The Beginning
-^^^^^^^^^^^^^^^^^^^^^^^
+Workflow
+^^^^^^^^
 
 Now, for the workflow! This is by no means the only way to perform this type of
-workflow, but I assume that you wish to handle conflicts as often as possible
-(so as to keep their total number small). Let us imagine that you have been at
-work, finished, and successfully pushed your changes to your *Origin*
-repository. You are now at home and want to continue working a bit. To begin,
-let's update our *home's local branches*::
-
-    .../cyclus_dir/$ git checkout main
-    .../cyclus_dir/$ git pull upstream main
-    .../cyclus_dir/$ git push origin main
-
-    .../cyclus_dir/$ git checkout work
-    .../cyclus_dir/$ git pull origin work
-    .../cyclus_dir/$ git rebase main
-    .../cyclus_dir/$ git push origin work
-
-Perhaps a little explanation is required. We first want to make sure that this new local copy of
-the main branch is up-to-date with respect to the remote origin's branch and remote upstream's
-branch. If there was a change from the remote upstream's branch, we want to push that to origin.
-We then follow the same process to update the work branch, except:
-
-#. we don't need to worry about the *upstream* repo because it doesn't have a work branch, and
-#. we want to incorporate any changes which may have been introduced in the main branch update.
-
-Workflow: The End
-^^^^^^^^^^^^^^^^^
-
-As time passes, you make some changes to files, and you commit those changes (to your *local work
-branch*). Eventually (hopefully) you come to a stopping point where you have finished your project
-on your work branch *AND* it compiles *AND* it runs input files correctly *AND* it passes all tests!
-Perhaps you have found Nirvana. In any case, you've performed the final commit to your work branch,
-so it's time to make a pull request online and wait for our main friends to
-review and accept it.
-
-Sometimes, your pull request will be held by the reviewer until further changes
-are made to appease the reviewer's concerns. This may be frustrating, but please
-act rationally, discuss the issues on the GitHub space made for your pull
-request, consult the `style guide
-<http://cyclus.github.com/devdoc/style_guide.html>`_, email the developer
-listhost for further advice, and make changes to your topic branch accordingly.
-The pull request will be updated with those changes when you push them to your
-fork.  When you think your request is ready for another review, you can reopen
-the review yourself with the button made available to you.
-
-See also
---------
-
-A good description of a git workflow with good graphics is available at
-http://nvie.com/posts/a-successful-git-branching-model/
+workflow, but we assume that you wish to handle conflicts as often as possible
```

Comment:
Wouldn't we want conflicts as little as possible? Maybe I don't understand what constitutes "handling a conflict"